3-enrich/stream-enrich/build.sbt

```diff
@@ -18,7 +18,8 @@ lazy val commonDependencies = Seq(
   Dependencies.Libraries.slf4j,
   Dependencies.Libraries.log4jOverSlf4j,
   Dependencies.Libraries.jacksonDatabind,
-  Dependencies.Libraries.s3Sdk,
+  Dependencies.Libraries.authSdk,
```

Comment:
added this explicitly instead of implicitly being included in the other libraries.